src/services/health/ipfs-cluster-utils.ts

```diff
@@ -1,34 +1,36 @@
-export const ndjsonParse = async function* (stream) {
-  const matcher = /\r?\n/;
-  const decoder = new TextDecoder('utf8');
-  let buffer = '';
+// TODO: what is this for? it isn't used in the code anywhere
```

Comment:
Not used? Delete it, but cc @vaultec81

Reply:
This is for parsing streamed results from IPFS cluster. Mainly relevant for getting a list of pins from the cluster and indexing locally. This should be kept for now as we need it for doing health checks and what not from the ipfs cluster.

Reply:
So there is work to do to integrate it? Since it's not referenced anywhere in the code

Reply:
I believe so. It is a separate scope of work to this PR

Reply:
Ok. Then, I think leaving it commented out is fine then @sisygoboom